### PR TITLE
Update build script

### PR DESCRIPTION
**Pull request by Mashintime**
Build an optimized binary with additional command line argument 'o'.  Otherwise acts the same way as previous iterations.

### CODE REVIEW

**Comment by Mashintime** [comment]
I think this read to merge into dev, if you all approve.

File: build [hunk 1]
```diff
@@ -40,7 +51,17 @@ gcc -g -o $F \
...
-# This addition is used to provide the application the capability to adjust the local system time -W2JON
+if [ $OPT -eq 1 ]; then
+	#Remove debugging stuff for a smaller binary
+	echo Stripping $F
+	strip $F
+fi
+
+
 echo "Brought to you by the sBitx 64 bit project team - W9JES, W2JON, KF7YDU, and N1QM" 
-sudo setcap CAP_SYS_TIME+ep /home/pi/sbitx/sbitx
```
Comment (drexjj) [inline build:45]: Is there an action to replace this?

**Comment by drexjj** [comment]
Does this display errors during the compile? If so, we need to hide the errors from the users..

File: build (hunk 1 above)
Comment (Mashintime) [reply build:45]: Yes, code block below.

**Comment by Mashintime** [comment]
> Does this display errors during the compile? If so, we need to hide the errors from the users..

A message will pop up saying to ignore warnings.  Users will have to take positive action (adding the o) to build an optimized binary, and I'd prefer to not hide the warnings as it could mask legitmate warnings.

File: build (hunk 1 above)
Comment (drexjj) [reply build:45]: yes sorry.

**Review by drexjj — APPROVED** [review]
Seems to work great.

**Pull request merged**